### ISSUE TITLE
Scancode Analyzer fails on calc demo

### ISSUE THREAD

**Issue by zetoph** [QMSTR/qmstr#460]
**Steps to reproduce:**
* checkout branch feat/packaging on endocode/qmstr
* checkout branch feat/packaging on endocode/qmstr-demo
* cd qmstr-demo/demos/compose/calc && docker-compose up -d --build --force-recreate 
* (view the logs via `docker logs -f calc_master_1`)
```
2020/02/12 15:15:04 scancode-analyzer failed with:
	--> 2020/02/12 15:13:16 Calling [scancode --full-root -l -c --json /home/qmstr/scancode.json --processes 8 /home/qmstr/buildroot/]
	--> * Configuring ScanCode for first use...
	--> Setup plugins...
	--> Collect file inventory...
	--> Scan files for: licenses, copyrights with 8 process(es)...
	--> Scanning done.
	--> Summary:        licenses, copyrights with 8 process(es)
	--> Errors count:   0
	--> Scan Speed:     2.13 files/sec. 
	--> Initial counts: 17 resource(s): 16 file(s) and 1 directorie(s) 
	--> Final counts:   17 resource(s): 16 file(s) and 1 directorie(s) 
	--> Timings:
	-->   scan_start: 2020-02-12T151349.962418
	-->   scan_end:   2020-02-12T151504.054489
	-->   setup_scan:licenses: 66.58s
	-->   setup: 66.58s
	-->   scan: 7.50s
	-->   total: 74.10s
	--> Removing temporary files...done.
	--> 2020/02/12 15:15:04 Analyzing file /home/qmstr/buildroot/add.c
	--> 2020/02/12 15:15:04 Found /home/qmstr/buildroot/add.c
	--> 2020/02/12 15:15:04 Found license information
	--> panic: interface conversion: interface {} is nil, not []interface {}
	--> 
	--> goroutine 1 [running]:
	--> main.(*ScancodeAnalyzer).detectCopyrights(0xc00009d4c0, 0xc000192020, 0x1b, 0x1, 0xc00017a038, 0x1)
	--> 	/go/qmstr/modules/analyzers/scancode-analyzer/main.go:243 +0xb01
	--> main.(*ScancodeAnalyzer).Analyze(0xc00009d4c0, 0xc000186060, 0x5eb7b56722bf397b, 0x0, 0x0)
	--> 	/go/qmstr/modules/analyzers/scancode-analyzer/main.go:94 +0x309
	--> github.com/QMSTR/qmstr/lib/go-qmstr/analysis.(*Analyzer).RunAnalyzerModule(0xc000186060, 0xc00009d4c0, 0xc000186060)
	--> 	/go/qmstr/lib/go-qmstr/analysis/analyzer.go:89 +0x78f
	--> main.main()
	--> 	/go/qmstr/modules/analyzers/scancode-analyzer/main.go:34 +0x54
```

**Comment by GiasemiSh** [comment]
You have changed the version of scancode. On master we use version 2.9.1. You are using 2.2.1. Please try scancode version 2.9.1

**Comment by zetoph** [comment]
I am currently using scancode-toolkit version 3.1.1
This appears to be the only available version for Python 3.X:
```
 % pip3 install scancode-toolkit==                                                                                                                                                                ~/Org wintermute
Collecting scancode-toolkit==
  Could not find a version that satisfies the requirement scancode-toolkit== (from versions: 3.1.1)
No matching distribution found for scancode-toolkit==
```

**Comment by zetoph** [comment]
This will be fixed by #464
